man/machinectl.xml

```diff
@@ -440,12 +443,15 @@
         environment variables or resource limits, among other
         properties.</para>
 
-        <para>Note that
-        <citerefentry><refentrytitle>systemd-run</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-        may be used in place of the <command>shell</command> command,
-        and allows more detailed, low-level configuration of the
-        invoked unit. However, it is frequently more privileged than
-        the <command>shell</command> command.</para></listitem>
+        <para>Note that <citerefentry><refentrytitle>systemd-run</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+        with its <option>--machine=</option> switch may be used in place of the <command>machinectl shell</command>
+        command, and allows non-interactive operation, more detailed and low-level configuration of the invoked unit,
+        as well as access to runtime and exit code/status information of the invoked shell process. In particular, use
+        <command>systemd-run</command>'s <option>--wait</option> switch to propagate exit status information of the
```

Comment:
I think we should use refs only when we first introduce a concept, once for each "section" of the man page, for a very blurry definition of section... Hence, when something appears twice, then the first one should be a ref, the second one a normal `<command>` and so on. Except if the mention of it is just brief and actually points people to the actual introduction, like in this case — where I think the best approach is to use a ref in the actual introduction, but a `<command>` in the first mention, if you follow what I mean...

Or to say this differently: I think it is less than ideal to reference something inside the reference to something else. Let the user instead follow the first ref, and then place any further ref in the text he followed too...

But I'd be very liberal with all this I must say...